packages/components/src/custom-gradient-picker/control-points.js

```diff
@@ -105,6 +105,7 @@ function ControlPointButton( {
 					color
 				) }
 				aria-describedby={ descriptionId }
+				aria-haspopup="true"
```

Comment:
I've chosen `true` as value instead of a more concrete one because 1) it's still valid as of ARIA 1.1 and 2) all the cases in which `aria-haspopup` is present we used this value so far. Happy to set it to a different value.

Reply:
`true` sounds fair to me, I can't see a better option.